src/enums/accountTypes.ts

```diff
@@ -0,0 +1,6 @@
+export enum AccountTypes {
+  PREMIUM = 'PREMIUM',
+  BASIC = 'BASIC',
+  STAFF = 'STAFF',
+  SBC_STAFF = 'SBC_STAFF'
+}
```

Comment:
I suggest these be moved to the shared repo.

Reply:
Good call.  I might need a quick refresher on the steps involved.   I'll reach out if I need help.